### PR TITLE
Fix data fetching

### LINKED ISSUE #28: Sample data cannot be fetched
Issue found upon running the following examples:

- road-surface-classification
- quarry-detection
- swimming-pool-detection/GE

### PR DESCRIPTION
**Pull request by acerioni**
Fixes #28

### CODE REVIEW

**Review by GwenaelleSa — APPROVED** [review]
I ran the process and it was working fine.
The metrics are quite low on the oth dataset. Did we adapt the number of steps and the selected model?

**Review by cleherny — COMMENTED** [review]
I have tested all the examples:
- The quarry project is working well
- The swimming pool projects are working well
- SWISSIMAGE dl is working again. Was there a problem at swisstopo?
- I have experienced an error with the script `prepare_data.py` for the road example:
image
Do you as well?

File: examples/quarry-detection/prepare_data.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 import morecantile
 import pandas as pd
 
-sys.path.insert(0, '.')
+sys.path.insert(0, '../..')
```
[the remark this answers is not in the document]
Comment (GwenaelleSa) [reply examples/quarry-detection/prepare_data.py:15]: Yes, I am working without Docker and both work for me too. Even not putting that line, which in my understanding is equivalent to `sys.path.insert(0, '.')`, works.
I don't get how finding path works in Python. I feel like every time I think I get it, something proves me wrong.

**Review by cleherny — APPROVED** [review]

File: examples/quarry-detection/prepare_data.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (cleherny) [reply examples/quarry-detection/prepare_data.py:15]: @acerioni, the previous version wasn't working on your side, isn't it?

**Pull request merged**